system-parachains/people/people-polkadot/src/lib.rs

```diff
@@ -110,9 +110,17 @@ pub type SignedExtra = (
 pub type UncheckedExtrinsic =
 	generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
 
+parameter_types! {
+	pub const IdentityMigratorPalletName: &'static str = "IdentityMigrator";
+}
+
 /// Migrations to apply on runtime upgrade.
 pub type Migrations = (
 	cumulus_pallet_xcmp_queue::migration::v5::MigrateV4ToV5<Runtime>,
+	frame_support::migrations::RemovePallet<
+		IdentityMigratorPalletName,
```

Comment:
Is this a bomb? how much data will it try and remove at once?

Reply:
The only state this has is `StorageVersion`.